reorder_python_imports/main.py

```diff
@@ -338,6 +349,13 @@ def main(argv=None):
             'application directories.  Defaults to `%(default)s`'
         ),
     )
+
+    parser.add_argument(
+        '--separate-relative', default=False, action='store_const', const=True,
+        help=(
+            'Put relative imports into bottom and seperate with other local '
+            'imports(absolute import) with an new line .'))
```

Comment:
please follow https://github.com/pre-commit/pre-commit/blob/master/CONTRIBUTING.md#multi-line-method-invocation for indentation